tests/Doctrine/Tests/ORM/Functional/Ticket/DDC3346Test.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Tests\Models\DDC3346\DDC3346Article;
+use Doctrine\Tests\Models\DDC3346\DDC3346Author;
+
+/**
+ * @group DDC-3346
+ */
+class DDC3346Test extends \Doctrine\Tests\OrmFunctionalTestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->setUpEntitySchema(
+            array(
+                'Doctrine\Tests\Models\DDC3346\DDC3346Article',
```

Comment:
Please use `const CLASSNAME = __CLASS__` in the referenced classes definitions: makes referencing classes much simpler and cleaner ;-)